### PR TITLE
8285366: Fix typos in serviceability

### PR DESCRIPTION
**Pull request by magicus**
I ran `codespell` on modules owned by the serviceability team (`java.instrument java.management.rmi java.management jdk.attach jdk.hotspot.agent jdk.internal.jvmstat jdk.jcmd jdk.jconsole jdk.jdi jdk.jdwp.agent jdk.jstatd jdk.management.agent jdk.management`), and accepted those changes where it indeed discovered real typos.

I will update copyright years using a script before pushing (otherwise like every second change would be a copyright update, making reviewing much harder).

The long term goal here is to make tooling support for running `codespell`. The trouble with automating this is of course all false positives. But before even trying to solve that issue, all true positives must be fixed. Hence this PR.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed (1 review required, with at least 1 reviewer)

### Issue
 * [JDK-8285366](https://bugs.openjdk.java.net/browse/JDK-8285366): Fix typos in serviceability

### Reviewers
 * [Kevin Walls](https://openjdk.java.net/census#kevinw) (@kevinjwalls - Committer) ⚠️ Review applies to [4b26668d](https://git.openjdk.java.net/jdk/pull/8334/files/4b26668d21b26eca90874abe5c3d7d38f59441cc)
 * [Serguei Spitsyn](https://openjdk.java.net/census#sspitsyn) (@sspitsyn - **Reviewer**) ⚠️ Review applies to [4b26668d](https://git.openjdk.java.net/jdk/pull/8334/files/4b26668d21b26eca90874abe5c3d7d38f59441cc)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/8334/head:pull/8334` \
`$ git checkout pull/8334`

Update a local copy of the PR: \
`$ git checkout pull/8334` \
`$ git pull https://git.openjdk.java.net/jdk pull/8334/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 8334`

View PR using the GUI difftool: \
`$ git pr show -t 8334`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/8334.diff">https://git.openjdk.java.net/jdk/pull/8334.diff</a>

</details>

### CODE REVIEW

File: src/jdk.jdwp.agent/share/native/libjdwp/invoker.h [hunk 1]
```diff
@@ -35,7 +35,7 @@
 typedef struct InvokeRequest {
     jboolean pending;      /* Is an invoke requested? */
     jboolean started;      /* Is an invoke happening? */
-    jboolean available;    /* Is the thread in an invokable state? */
+    jboolean available;    /* Is the thread in an invocable state? */
```
Comment (kevinjwalls) [inline src/jdk.jdwp.agent/share/native/libjdwp/invoker.h:38]: invocable could perhaps stay as invokable ?
Elsewhere we have a filename com/sun/tools/jdi/InvokableTypeImpl.java which we clearly don't want to change,  and I see Internet dictionary evidence of invokable being acceptable.

**Review by kevinjwalls — APPROVED** [review]
All looks good to me, just the invokable which you might want to leave as is, unless there are other strong feelings. 8-)

**Comment by dfuch** [comment]
> @dfuch I have only updated files in `src`, so if the incorrect spelling is tested, that test will now fail. I'm unfortunately not well versed in what tests cover serviceability code. Can you suggest a suitable set of tests to run?

Folks from serviceability-dev will be more able to answer that - but I would suggest running tier2-tier3 as a precaution.

File: src/jdk.jdwp.agent/share/native/libjdwp/invoker.h (hunk 1 above)
Comment (kevinjwalls) [reply src/jdk.jdwp.agent/share/native/libjdwp/invoker.h:38]: Sure, I just thought there was enough evidence that invokable is not definitely wrong, even if invocable is more correct and popular, so it's not obvious it should be changed.  Don't lose sleep over it. 8-)

More importantly, on the tests -- I see the changes in exception messages searched for the wrong text in the test directories, and didn't find any matches that looked like checks.

**Comment by plummercj** [comment]
> > @dfuch I have only updated files in `src`, so if the incorrect spelling is tested, that test will now fail. I'm unfortunately not well versed in what tests cover serviceability code. Can you suggest a suitable set of tests to run?
> 
> Folks from serviceability-dev will be more able to answer that - but I would suggest running tier2-tier3 as a precaution.

I sent Magnus a PM with info on all the svc tests that can be run.

**Review by sspitsyn — APPROVED** [review]
Hi Magnus,
Great job, thank you for doing this!
It looks good to me.
These findings are pretty interesting.
Some are funny and some seems to be typical.
Good examples of typical typos are:
 - double letter instead of single
 - single or even triple letter instead of double
 
 Thanks,
Serguei

File: src/jdk.jdwp.agent/share/native/libjdwp/invoker.h (hunk 1 above)
Comment (dholmes-ora) [reply src/jdk.jdwp.agent/share/native/libjdwp/invoker.h:38]: Invocable, Invokable and Invokeable are all used in practice. We have a mix of invocable and invokable throughout our codebase, with more of the former than the latter - and in prose "invocable" is probably best.

**Pull request closed without merging**